org.sugarlabs.ReadETexts.json

```diff
@@ -283,18 +254,6 @@
                 {
                     "type": "patch",
                     "path": "readetexts-info.patch"
-                },
-                {
-                    "type": "patch",
-                    "path": "readetexts-fix-save.patch"
-                },
-                {
-                    "type": "patch",
-                    "path": "readetexts-fix-print.patch"
-                },
-                {
-                    "type": "patch",
-                    "path": "readetexts-fix-pango.patch"
                 }
             ],
             "post-install": [
```

Comment:
in post-install, no need for "python3 /app/bin/" , since these utils are already python3.